ClinGen/clincoded
# Connects to #1328. Preparation for loading new disease data into instance and running upgrade scripts.

**Technical notes:**
1. This PR consists of changing the `id` property name in the `disease` object schema to `diseaseId`.
2. As a result, various JS and schema files that have references to the disease `id` property are also updated to `diseaseId'.
3. Various upgrade scripts are updated to migrate the current Orphanet record UUIDs to the mapped/equivalent MonDO disease record UUIDs.

**Steps to test:**
1. Spin up an instance with this branch and production data.
2. Run Python script to insert MonDO disease records into the `disease` object in ElasticSearch.
3. Run the upgrade scripts manually to migrate the diseases.
4. Wait reindexing to complete.
5. Review different types of records (e.g. GDM, Group, Family, Individual, and Interpretation) to verify whether migrated diseases are matching those in production currently.
